plugins/UsersManager/TokenNotifications/TokenNotificationInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\UsersManager\TokenNotifications;
+
+interface TokenNotificationInterface
+{
+    public function getTokenId(): string;
+
+    public function getTokenName(): string;
+
+    public function getTokenCreationDate(): string;
+
+    public function dispatch(): bool;
```

Comment:
The `TokenNotifierTask` is currently ignoring the return value.

Would it make sense to at least add a "if dispatch returned fales, do not set notified"? Or can we change this to `void` if we don't intend to act upon it?